From 86c52ed0945f7c9fca7f7e596b27bc8c1d64f36b Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Wed, 1 Nov 2017 17:28:31 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: make rk3368 cru syscon"

This reverts commit bfd8339bc256b9a3c01972ce6a6570dd8ee47682.
cif driver don't access cru directly.

Change-Id: I907b700f1792be5ea7c4127638f55782f6223d00
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 1d81a1fbda3a..2b561330ed54 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -1136,7 +1136,7 @@
 	};
 
 	cru: clock-controller@ff760000 {
-		compatible = "rockchip,rk3368-cru", "syscon";
+		compatible = "rockchip,rk3368-cru";
 		reg = <0x0 0xff760000 0x0 0x1000>;
 		rockchip,grf = <&grf>;
 		#clock-cells = <1>;
-- 
2.35.3

